src/commands/api/uploadAppSettingsFromApi.ts

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+import { callWithTelemetryAndErrorHandling, IActionContext } from "vscode-azureextensionui";
+import { ISimpleAppSettingsClient } from "../../vscode-azurefunctions.api";
+import { uploadAppSettingsInternal } from "../appSettings/uploadAppSettings";
+
+export async function uploadAppSettingsFromApi(client: ISimpleAppSettingsClient, workspacePath?: string): Promise<void> {
```

Comment:
Let's remove `workspacePath`. I don't see it used in SWA or even listed in "vscode-azurefunctions.api.d.ts"

Reply:
Yeah, I meant to remove that since Functions has logic regarding retrieving the workspacePath if not provided 👍